### PR TITLE
Add default preview image as an setting (closes #246)

### LINKED ISSUE #246: Add default preview image as an setting
* `FRIGG_PREVIEW_IMAGE`
* Should be used if image is not specified in preview settings.

### PR DESCRIPTION
**Pull request by frecar**
@relekang please review :)

### CODE REVIEW

File: frigg/deployments/models.py [hunk 1]
```diff
@@ -52,10 +52,16 @@ def is_pending(self):
     @property
     def queue_object(self):
         obj = self.build.queue_object
+
+        if self.image and self.image != "":
```
Comment (relekang) [inline frigg/deployments/models.py]: Is this change necessary? As self.image is already [set correctly](https://github.com/frigg/frigg-hq/pull/250/files#diff-01d4d2de9b7ba8519ce768d37eae5c81R278).

Comment (frecar) [inline frigg/deployments/models.py]: well, we may add more ways to create a PRDeployment-object later, and this will ensure that this is always the behavior we are expecting ?

**Comment by relekang** [comment]
:shipit:
